onnxruntime/core/providers/nnapi/nnapi_builtin/nnapi_lib/NeuralNetworksWrapper.h

```diff
@@ -99,12 +102,40 @@ inline std::string TypeToStr(const Type& type) {
   }
 }
 
+struct SymmPerChannelQuantParams {
```

Comment:
This is from AOSP https://android.googlesource.com/platform/frameworks/ml/+/refs/heads/master/nn/runtime/include/NeuralNetworksWrapper.h

Keep the style of the source file